Aptoide/aptoide-client-v8
# [ASV-1068] AppNext AB Test

**What does this PR do?**

  Adds an AB Test experiment for the ad that appears on similar apps inside an appview. The first case is the default (old) one, where the ad is retrieved from our webservices, and it accounts for 90% of the cases. The other case is the new appnext ad (10%).

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] AdsRepository.java
- [ ] SimilarAdExperiment.java

**How should this be manually tested?**

  Check if each case is working normally
  1st case (90%): Old ad should show up in similar apps
  2nd case (10%): New ad should show up in similar apps instead.

The test case the device gets attributed to can be changed by manually rewriting the response in Charles, or by changing the variable value in debug.

**Note:** APPNEXT_PLACEMENT_ID needs to be set in the global gradle.properties. The value can be found [here](https://aptoide.atlassian.net/browse/ASV-1057).

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1068](https://aptoide.atlassian.net/browse/ASV-1068)

**Other notes:**

   The way this experiment is setup is a bit different from the other active ones, in the sense that it's not using a Boolean to return the experiment assigment, but rather directly returning the relevant value to the chain.




**Code Review Checklist**
